lib/core/connector_settings.rb

```diff
@@ -19,6 +19,9 @@ class ConnectorSettings
     DEFAULT_REDUCE_WHITESPACE = true
     DEFAULT_RUN_ML_INFERENCE = true
 
+    DEFAULT_FILTERING = {}
+    DEFAULT_ADVANCED_FILTER_CONFIG = {}
```

Comment:
These should not be constants IMO

Reply:
I think we should keep it concise with the rest of the class, which also specifies simple boolean values (f.e. `DEFAULT_REDUCE_WHITESPACE`) as constants.